src/key_value/key_value_interface.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+
+#ifndef SRC_KEY_VALUE_KEY_VALUE_INTERFACE_H_
+#define SRC_KEY_VALUE_KEY_VALUE_INTERFACE_H_
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace csci499 {
+
+// key value storage for csci499 application
+class KeyValueInterface {
```

Comment:
Could you add more descriptive comments for the class and interfaces?  Especially explain why the interface exists.